src/main/java/com/craftaro/ultimateclaims/claim/Claim.java

```diff
@@ -91,7 +85,11 @@ public int getId() {
     }
 
     public ClaimedChunk getFirstClaimedChunk() {
-        return this.claimedRegions.iterator().next().getFirstClaimedChunk();
+        try {
+            return this.claimedRegions.iterator().next().getFirstClaimedChunk();
+        } catch (NoSuchElementException handled) {
+            return null;
+        }
```

Comment:
If it is possible to not use a try-catch I'd prefer that a lot.

In this case the `NoSuchElementException` could be prevented by checking if there are any `claimedRegions` available before calling `#next()` or if possible even before calling `#iterator()`.

With this we have a clear logic that can be understood later:
if no claimed regions => return null
else => return first claimed region